src/foam/dao/java/JDAO.js

```diff
@@ -32,11 +37,17 @@ foam.CLASS({
               .setCreateFile(true)
               .build());
 
-            // create a composite journal of repo journal
-            // and runtime journal and load them all
+            /* Create a composite journal of repo journal and runtime journal
+              and load them all.*/
+            X resourceStorageX = x;
+            if ( System.getProperty("RESOURCES_JOURNAL_SOURCE") != null ) {
+              resourceStorageX = x.put(foam.nanos.fs.Storage.class,
+                  new Storage(System.getProperty("RESOURCES_JOURNAL_SOURCE"), true));
+            }
+
```

Comment:
Can RESOURCE_JOURNAL_SOURCE property be not set?  What happens then? Or does it matter -- just assume its set?